supervisory-authorities/derfswr.json

```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "derfswr",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Rundfunkbeauftragter für den Datenschutz beim Südwestrundfunk",
+    "address": "Neckarstraße 230\n70190 Stuttgart\nDeutschland",
+    "phone": "+49 711 92913014",
+    "fax": "+49 711 92913019",
+    "email": "datenschutz@SWR.de",
+    "web": "https://www.swr.de/",
+    "sources": [
+        "https://www.swr.de/unternehmen/organisation/datenschutz-128.html",
+        "https://www.bfdi.bund.de/SharedDocs/Adressen/DE/Rundfunkdatenschutzbeauftragte/DatenschutzbeauftragterSuedwestrundfunk.html"
```

Comment:
Better source: https://www.swr.de/unternehmen/organisation/datenschutz-128.html

Reply:
This source is already listed. Do you mean I should remove the second source?

Reply:
Oh sorry, missed that.

As for removing the BfDI one: Usually, we only add "necessary" sources (i.e. ones that cover details not covered by the other ones). I'll let you decide in this case, I guess it could make sense to leave the BfDI for every public broadcasting SVA.

Reply:
I would leave it in for now and possible future reference. We could still remove it in the future. Or is there anything connected with the sources, e.g., automatic tasks, that would benefit from fewer sources = less work? Then, removing it would be the better option in my opinion.

Reply:
No, that's fine. I am working on a script to automatically verify that all data in the records is still up-to-date but an additional source doesn't matter for that.